### DISCUSSION TITLE
No longer able to manually set download location/filename?

### DISCUSSION THREAD

**Discussion by Globoxthe2** [aclap-dev/video-downloadhelper#117] [Help and Q&A]
### Your question

I have been using VDH for a while now to download clips from Twitch to make compilations around once a week, but between tonight and last week something seems to have changed?
Normally, my process is to open the page for a clip I want, click the three ball icon, mouse over the quality I want, click the -> arrow, click download, navigate to the folder for a specific project, name it based on game/stream/credit for who made the clip, and hit enter to start the download. Wash rinse repeat.
Now when I am on a page, click the -> next to the file/quality I want and click "Download", instead of asking for the destination/file name as usual, it does the exact same thing as "Quick download": Immediately downloads it to my browser's default download directory, and doesn't let me rename it. 

I haven't changed any settings for as far back as I can remember, except for putting some blacklisted URLs back as an update seems to have wiped my blacklist out at some point? I can't seem to find any settings that would have impacted this either.

I can still technically function with the "Download" button not working the way it should, but it seriously impacts my process as now I have a bunch of unnamed, unorganized, unattributed video files that I'll have to spend a bunch of time sorting and renaming after the fact which is a huge pain in the butt. Help?

### Addon & CoApp version and details

Firefox, V-8.2.0.23 without CoApp. Windows 10 x64

**Comment by paulrouget** [comment]
Oh, that would explain some other issues we have. Let me look into it.

**Reply by Globoxthe2** [reply]
Thanks for the quick response! I'll admit that I'm probably using it in a lot less effective/efficient way than I could be, but if that means my problem now helps you sort out some other stuff easier then I'm glad I swung by to ask. 😄 Fingers crossed it's not too much of a pain to sort out for you.

**Comment by paulrouget** [comment]
My bad. I checked. Totally my fault.

I'll fix that in v8.2.0.26. Might take a day or two.

Thank you for the detailed (and polite) bug report :)
